staging/src/k8s.io/apiserver/pkg/util/flag/map_string_bool.go

```diff
@@ -34,6 +34,9 @@ type MapStringBool struct {
 // NewMapStringBool takes a pointer to a map[string]string and returns the
 // MapStringBool flag parsing shim for that map
 func NewMapStringBool(m *map[string]bool) *MapStringBool {
+	if *m == nil {
```

Comment:
leaving this nil until Set is called for the first time was intentional (difference between empty (`--foo=`) and unset (no `--foo` flag at all)

if this is just to fix e2e help, can we initialize the flag correctly there?

Reply:
This can not fix e2e help panic